guides/deploy/starter_deploy_kubernetes.html

```diff
@@ -0,0 +1,83 @@
+<p>You need to build the Docker image that will run your Kitura application and will deployed to your Kubernetes cluster. The following commands should all be run from the directory containing your Dockerfiles.</p>
+<p>First, build the image that will be used to compile your app.</p>
+<div><pre><code class="sourceCode bash">docker build -t myapp-build -f Dockerfile-tools .</code></pre></div>
+<p>Then, use this image to compile your application code into an executable:</p>
+<div><pre><code class="sourceCode bash">docker run -v $PWD:/swift-project -w /swift-project myapp-build /swift-utils/tools-utils.sh build release</code></pre></div>
+<p>You’ll need to use the default Dockerfile to build the image that will run the executable you just created:</p>
+<div><pre><code class="sourceCode bash">docker build -t myapp-run .</code></pre></div>
+<p>You can check that your container runs correctly by running the following command and then visiting <code>localhost:8080</code> in your browser:</p>
+<div><pre><code class="sourceCode bash">docker run -p 8080:8080 -it myapp-run</code></pre></div>
+<p>Once you are happy with your Docker image, tag it appropriately for the registry you are using and push it. If you are yet to choose a registry and have no specific cloud provider requirements, we suggest you try <a href="https://hub.docker.com/">Docker Hub</a>.</p>
+<h4>Deploy to Kubernetes</h4>
+<p>Before you deploy your application to your Kubernetes cluster, you need to make sure that the Helm chart is referencing the Docker image you’ve just pushed to your registry. You do this by editing the <code>repository</code> variable in the <code>values.yaml</code> file in your <code>/chart</code>directory.</p>
+<p>The Helm chart provided configures a number of default resource options, such as memory allowance and replica policies. You can make changes to this depending on your or your cloud provider’s requirements.</p>
+<p>To install the chart into your cluster, run the following from the <code>/chart/&lt;your app name&gt;</code> directory:</p>
+<div><pre><code class="sourceCode bash">helm install --name myapp .</code></pre></div>
+<p>You may have to wait a few minutes for it to be available, but your application should now be deployed to your Kubernetes cluster. To access your application, you have to find the external IP of the worker node it has been deployed to. This process varies, so check with your cloud provider on how to do so. The port that your application had been deployed on is described by the <code>NodePort</code> which you can find by running the following command on your cluster:</p>
+<div><pre><code class="sourceCode bash">kubectl describe service &lt;your app name&gt;-service</code></pre></div>
+<p>Congratulations! You have now successfully deployed your Kitura application using Kubernetes! If you visit <code>&lt;external IP&gt;:&lt;NodePort&gt;</code> you should se the Kitura splash page.</p>
```

Comment:
"you should se" -> "you should see"

Also this code tag looks like straight text <code>&lt;external IP&gt;:&lt;NodePort&gt;</code> I suggest maybe changing it to <highlight>

Reply:
For now lets just highlight them, you can do this by wrapping the text in `<span>` tags like so: 
`<span class="highlight">Text you want highlighting</span>`

When I move onto refactoring the all of the guides styling/layout, I'll be addressing the this styling as well and will update it to whatever we decide.